packages/storybook-react/src/stories/Paragraph.stories.tsx

```diff
@@ -1,22 +1,18 @@
 import readme from '@nl-design-system-unstable/documentation/componenten/_paragraph.md?raw';
 import { Meta, StoryObj } from '@storybook/react';
-import { Paragraph } from '@utrecht/component-library-react/dist/css-module/index';
+import { Paragraph } from '@utrecht/component-library-react';
 import tokensDefinition from '@utrecht/components/paragraph/tokens.json';
 import tokens from '@utrecht/design-tokens/dist/index.json';
-import React from 'react';
-import { designTokenStory } from './util';
 
-const meta = {
+export const Default: Meta<typeof Paragraph> = {
   title: 'React Component/Paragraph',
-  id: 'react-paragraph',
```

Comment:
De `id` properties voegen we handmatig toe volgens een voorspelbaar patroon, zodat we ze in een iframe kunnen includen in docusaurus. Bijvoorbeeld: https://nl-design-system.github.io/utrecht/docs/componenten/typografie/paragraaf/

We moeten hier eigenlijk wat documentatie voor hebben.